src/core/validators/validators.ts

```diff
@@ -57,7 +59,7 @@ const MinLength = (minlength: number): ValidatorFn => {
  */
 const MaxLength = (maxLength: number): ValidatorFn => {
   return (value) => {
-    if (!value|| maxLength >= value.length) {
+    if (!value|| maxLength >= value.length || Object.getPrototypeOf(value) === Object.prototype) {
```

Comment:
you could use lodash isEmpty for all these checks imo

Reply:
Here, what I'm checking is that is an object strict. If I use the lodash ._isObject() it does not differentiate [] from {}